### ISSUE TITLE
PyTorch Geometric for Node Regression

### ISSUE THREAD

**Issue by bianchiniriccardo95** [pyg-team/pytorch_geometric#3794]
Hello! First of all, thank you for this amazing tool.
I have one question: can I use PyTorch Geometric to perform node regression? If so, how can I implement the GCN to perform this task?
Thanks in advance.

**Comment by rusty1s** [comment]
Performing node regression is similar to performing node classification, except that you use a different loss formulation, e.g., `torch.nn.functional.mse_loss`. That is, your GNN now maps to a single scalar output value per node, and usess `mse_loss`:
```python
out = model(data.x, data.edge_index)  # shape [num_nodes, 1]
loss = F.mse_loss(out.squeeze(), data.y.squeeze())
```

**Comment by bianchiniriccardo95** [comment]
Thank you very much for your quick reply!

**Comment by SourabhRanade** [comment]
@bianchiniriccardo95 by any chance do you have any examples that I can look into to perform node regression?
